src/status_im/hardwallet/core.cljs

```diff
@@ -8,31 +8,75 @@
             [taoensso.timbre :as log]
             [status-im.i18n :as i18n]
             [status-im.accounts.create.core :as accounts.create]
-            [status-im.accounts.login.core :as accounts.login]
             [status-im.node.core :as node]))
 
+(defonce default-pin "000000")
+
 (defn hardwallet-supported? [{:keys [db]}]
   (and config/hardwallet-enabled?
        platform/android?
        (get-in db [:hardwallet :nfc-supported?])))
 
+(fx/defn unauthorized-operation
+  [{:keys [db] :as cofx}]
+  (fx/merge cofx
+            {:db (assoc-in db [:hardwallet :on-card-connected] nil)
+             :utils/show-popup {:title   ""
```

Comment:
Maybe add a title?

Reply:
@jeluard i need UX and QA to look into this during testing first, then we can come with some meaningful title